servers/scripts/ddnet.py

```diff
@@ -470,7 +470,7 @@ def printPlayers(server, filt, con, cur):
       else:
         htmlName = escape(player['name'])
 
-      print((u"  <td class=\"time\">%s</td><td class=\"name\">%s</td><td class=\"clan\">%s</td><td class=\"flag\"><img src=\"countryflags/%s.png\" alt=\"%s\" height=\"20\"/></td>" % (formatScore(player['score'], "race" not in server['game_type'].lower()), htmlName, escape(player['clan']), countryFlags.get(player['country'], 'default'), countryFlags.get(player['country'], 'NONE'))).encode('utf-8'))
+      print((u"  <td class=\"time\">%s</td><td class=\"team\">Team %s</td><td class=\"name\">%s</td><td class=\"clan\">%s</td><td class=\"flag\"><img src=\"countryflags/%s.png\" alt=\"%s\" height=\"20\"/></td>" % (formatScore(player['score'], "race" not in server['game_type'].lower()), escape(player["team"]), htmlName, escape(player['clan']), countryFlags.get(player['country'], 'default'), countryFlags.get(player['country'], 'NONE'))).encode('utf-8'))
```

Comment:
Is that really useful? Most will show team 0 anyway. Will take a lot of space

Reply:
probs not, maybe we could color them? and the team 0 color is just invisible

Reply:
Colored text would look bad, background color would be ok.